catalog/pipeline/release/0.4/README.md

```diff
@@ -0,0 +1,37 @@
+# Milestone-9 Release Pipeline
+
+Tekton pipeline to release HACBS Snapshot to Quay.
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | The Snapshot in JSON format | No | - |
+| enterpriseContractPolicy | JSON representation of the policy to be applied when validating the enterprise contract | No | - |
+| extraConfigGitUrl |URL to the remote Git repository containing the extra config | No | - |
+| extraConfigGitRevision | Revision to fetch from the remote Git repository containing the extra config | No | - |
+| extraConfigPath | Path to the extra config file within the repository | No | - |
+| postCleanUp | Cleans up workspace after finishing executing the pipeline | Yes | true |
+
+
```

Comment:
We need a `Changes since 0.3` section saying that `Parameter applicationSnapshot was changed to snapshot`